drivers/bluetooth/broadcom/v4l2_fm_driver/fmdrv_config.h

```diff
@@ -46,9 +46,11 @@
 
 /* Set default Audio path */
 #ifndef DEF_V4L2_FM_AUDIO_PATH
```

Comment:
Confirmed!
Shinano uses DAC
So the change would be something like this:

> #ifndef CONFIG_MACH_SONY_SHINANO
>     #define DEF_V4L2_FM_AUDIO_PATH FM_AUDIO_I2S
> #else
>     #define DEF_V4L2_FM_AUDIO_PATH FM_AUDIO_DAC
> #endif

Good work!
Happy 2017!